src/pages/admin-ui/base.page.ts

```diff
@@ -2,7 +2,7 @@ import { Locator, Page } from '@playwright/test';
 import { ConfigurationPage, ContentManagementPage, DashboardPage, HomePage, LoginPage, ScheduledEventPage, UserPage } from '.';
 
 export class BasePage {
-  private readonly page: Page;
+  readonly page: Page;
```

Comment:
Yes it shouldn't be private, but protected I believe so child classes can use it